### ISSUE TITLE
Python (and likely others): 'filters' parameter in list_documents should be 'filters[]'

### ISSUE THREAD

**Issue by peterjcarroll** [appwrite/sdk-generator#46]
I was attempting to use the Python SDK to call the list_documents method on the Database class using a filter, like so:

```
db = Database(client)
teachers = db.list_collections('Teachers')['collections'][0]
english_teachers = db.list_documents(teachers['$id'], filters=['name=Bilbo Baggins'])
```

This gave me a 400 Bad Request error:
```
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: http://localhost:2506/v1/database/collections/5ef580dd8f311/documents?filters=name%3DBilbo+Baggins&offset=0&limit=50&orderField=%24id&orderType=ASC&orderCast=string&search=&first=0&last=0
```

The body of that error response was this:
`Invalid filters: Value must be an array and Value must be a string and no longer than 128 chars`

I'm not a PHP guy so the solution was not obvious to me. Eventually I found this link which explains how to pass an array to PHP via the query string: https://stackoverflow.com/questions/6243051/how-to-pass-an-array-within-a-query-string

For now I have a fork of the Python SDK which works by changing the name of the 'filters' parameter in list_documents to 'filters[]': https://github.com/peterjcarroll/sdk-for-python/commit/b9757a9b9cffa9672d9d3bd2a9cddc216880e780

But since I know that the SDKs are generated, the solution is to fix the generator.

I took a look at some of the other SDKs for other languages, and I think this fix will need to be applied to those as well.

I hope this is enough information, let me know if you need anything else to get this fixed. Thanks!

**Comment by eldadfux** [comment]
Thanks for reporting this issue, as most language aren't really fans of the "[]" syntax, I am also thinking of adding 'regular' array support directly in the Appwrite API. I'll post updates to this issue if there are any on the API part as well.

**Comment by eldadfux** [comment]
@peterjcarroll this issue should now be resolved in version 0.0.6 of the Python SDK.
